themes/bootstrap3/templates/Recommend/TopFacets.phtml

```diff
@@ -55,7 +54,8 @@
 <?php
 $script = <<<JS
 VuFind.truncate.initTruncate('.top-facets-contents');
-collapseTopFacets();
+// Close with JS (ensures content is shown to non-JS users)
+$(this).find('.collapse').removeClass('in');
```

Comment:
Should this change, and the removal of collapseTopFacets(), be a separate PR? It doesn't seem to have anything to do with icons, but maybe I'm missing something.

Reply:
`collapseTopFacets` was toggling this class and changing the icon. The icon helper automatically changes the icon based on CSS, so the function became obsolete when we used `$this->icon('collapse');`.